### PR TITLE
rg: update title to match page name; improve examples

### PR DESCRIPTION
**Pull request by bl-ue**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5071
cc @MasterOdin

### CODE REVIEW

File: pages/common/rg.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# ripgrep
+# rg
 
-> A recursive line-oriented CLI search tool.
+> Ripgrep (rg) is a recursive line-oriented CLI search tool.
```
Comment (bl-ue) [inline pages/common/rg.md:3]: @navarroaxel `ripgrep` or `Ripgrep` here?

Comment (navarroaxel) [reply pages/common/rg.md:3]: Ripgrep

Comment (navarroaxel) [reply pages/common/rg.md:3]: Ripgrep

Comment (navarroaxel) [reply pages/common/rg.md:3]:
```suggestion
> Ripgrep is a recursive line-oriented CLI search tool.
```

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

**Pull request merged**